src/together/resources/finetune.py

```diff
@@ -614,12 +614,12 @@ def download(
                 )
             url += "&checkpoint=model_output_path"
         elif isinstance(ft_job.training_type, LoRATrainingType):
-            if checkpoint_type == DownloadCheckpointType.DEFAULT:
-                checkpoint_type = DownloadCheckpointType.MERGED
+            if checkpoint_type == DownloadCheckpointType.DEFAULT.value:
```

Comment:
I think that a safer way would be to make the function accept both strings and `DownloadCheckpointType`, and cast the string to `DownloadCheckpointType` at the beginning of the method. This way, we will be able to compare with Enum values further in the code while still guaranteeing a fixed range of possible inputs